### PR TITLE
feat: add validate cli

### PR DESCRIPTION
**Pull request by sqin2019**

### CODE REVIEW

File: pkg/requestutil/request.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/abcxyz/access-on-demand/apis/v1alpha1"
+	"gopkg.in/yaml.v3"
+)
+
+func ReadFromPath(path string) (*v1alpha1.IAMRequest, error) {
```
Comment (sethvargo) [inline pkg/requestutil/request.go:27]: Doc

**Review by sethvargo — APPROVED** [review]

File: pkg/requestutil/request.go (hunk 1 above)
Comment (sqin2019) [reply pkg/requestutil/request.go:27]: Oops, updating!

**Comment by sethvargo** [comment]
@sqin2019 if you'd like, you can:

```go
go get -u github.com/abcxyz/pkg@main
```

and then add tab completions. This will be really helpful for browsing to files (you can use `predict.Files` to have tab completion for the local filesystem). See https://github.com/abcxyz/pkg/pull/125 for some examples.

**Comment by sqin2019** [comment]
> @sqin2019 if you'd like, you can:
> 
> ```go
> go get -u github.com/abcxyz/pkg@main
> ```
> 
> and then add tab completions. This will be really helpful for browsing to files (you can use `predict.Files` to have tab completion for the local filesystem). See [abcxyz/pkg#125](https://github.com/abcxyz/pkg/pull/125) for some examples.
> 
> This can also be a follow-up if you want. We just added this functionality, so it's not a blocker.

Thanks, will check it out and follow up!

**Pull request merged**